.evergreen/config.yml

```
@@ -1075,6 +1128,14 @@ buildvariants:
            - name: "test-2.4-sharded_cluster"
            - name: "test-2.4-standalone"
 
+- matrix_name: "serverless"
+  matrix_spec:
+    os-requires-32:
+      - "ubuntu-16.04"
+  display_name: "Serverless ${os-fully-featured}"
```

If I am understanding the matrix definition, I think this should be:
```suggestion
  display_name: "Serverless ${os-requires-32}"
```

---

Since we're now using 18.04 I updated this to `os-requires-40`.